prow/jobs/incubator/vstudio-extension/vstudio-extension.yaml

```diff
@@ -0,0 +1,45 @@
+job_template: &job_template
+  skip_report: true
+  decorate: true
+  path_alias: github.com/kyma-incubator/vstudio-extension
+  max_concurrency: 10
+  extra_refs:
+    - org: kyma-project
+      repo: test-infra
+      base_ref: master
+      path_alias: github.com/kyma-project/test-infra
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/prow/test-infra/buildpack-node:v20181130-b28250b
+        securityContext:
+          privileged: true
+        command:
+          - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build.sh"
+        args:
+          - "/home/prow/go/src/github.com/kyma-incubator/vstudio-extension"
+
+job_labels_template: &job_labels_template
+  preset-dind-enabled: "true"
+  preset-sa-gcr-push: "true"
+  preset-docker-push-repository-incubator: "true"
+
+presubmits: # runs on PRs
+  kyma-incubator/vstudio-extension:
+    - name: vstudio-extension
+      always_run: true
+      branches:
+        - master
+      <<: *job_template
+      labels:
+        <<: *job_labels_template
+        preset-build-pr: "true"
+
+postsubmits:
+  kyma-incubator/vstudio-extension:
+    - name: vstudio-extension
```

Comment:
should we define always_run for postsubmits?

Reply:
No, it is a property of presubmit job type: https://github.com/kubernetes/test-infra/blob/050e7f5e2cd684ce12e9d382363c84f2dcdcd7ab/prow/config/jobs.go#L118